### ISSUE TITLE
Bug: TypeError: this.cliEngine is not a constructor

### ISSUE THREAD

**Issue by Manuel-Suarez-Abascal** [eslint/eslint#15175]
### Environment

Node version: v14.17.0
npm version: 6.14.13
Local ESLint version: 8.0.1
Operating System: macOS Big Sur

### What parser are you using?

vue-eslint-parser

### What did you do?

I'm using [Vue CLI](https://cli.vuejs.org/) & after upgrading `ESlint` from version `7.32.0` to `8.0.1` I get the error `Syntax Error: TypeError: eslint.CLIEngine is not a constructor` after launching the project for development with `npm run serve`.

```js
// .eslintrc.js
module.exports = {
  root: true,
  env: {
    node: true
  },
  extends: ["plugin:vue/essential", "@vue/prettier"],
  rules: {
    "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
    "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off"
  },
  parserOptions: {
    parser: "babel-eslint"
  }
};



### What did you expect to happen?

Before upgrading `ESlint`, I was able to launch the project for development without issues.

### What actually happened?

I got this error: `Syntax Error: TypeError: eslint.CLIEngine is not a constructor` & my code won't compile for development.

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

_No response_

**Comment by shavkat1121** [comment]
whats'up?

**Comment by MCYouks** [comment]
Any news?

**Comment by argahsuknesib** [comment]
what should we do?

**Comment by aladdin-add** [comment]
there is nothing we can do in eslint, please follow the linked vue's issue to get latest news.

**Comment by cakcak27** [comment]
i'm solved with installing eslint version 6.2.0 in dev env

**Comment by liuyanhe96** [comment]
@cakcak27 thanks for u reply

**Comment by shamsail** [comment]
And it's still happening with v8.
So, I have downgraded and using this combination.

```
"eslint": "^6.7.2",
"eslint-plugin-vue": "^6.2.2",
```

It will work for others as well.

**Comment by dprez8** [comment]
@shamsail in which file please??

**Comment by shamsail** [comment]
> @shamsail in which file please??

Edit: change these into package.json file and to install properly, first remove/delete node_modules and package_lock.json file and npm install again after updating package.json file

**Comment by UtmostCreator** [comment]
vue vertion `^2.6.11`
1. Stop your application;
2. Update the versions by running `npm i`;
Running excellent along with:
```
    "eslint": "^6.8.0",
    "eslint-plugin-vue": "^6.2.2",
```
3. start/run your application again

**Comment by EdgarAlonsoPerez** [comment]
Hi guys  i've tried some options but this one worked for me.
I hope this can help someone :D 

```"devDependencies": {
        "@vue/cli-plugin-babel": "^3.3.0",
        "@vue/cli-plugin-eslint": "^3.1.1",
        "@vue/cli-service": "^3.12.1",
        "@vue/compiler-sfc": "^3.0.0",
        "babel-eslint": "^10.1.0",
        "eslint": "^6.2.0",
        "eslint-plugin-vue": "^7.0.0"
    }```

**Comment by eraloop** [comment]
Hello guys, i kindof have thesame issue, have tried deleting node modules, updating eslint, but to no avail. here is my dev dependencies.
`
  "devDependencies":  {
    "@vue/cli-plugin-babel": "~4.5.0",
    "@vue/cli-plugin-eslint": "~4.5.0",
    "@vue/cli-plugin-pwa": "~4.5.0",
    "@vue/cli-plugin-router": "~4.5.0",
    "@vue/cli-plugin-vuex": "~4.5.0",
    "@vue/cli-service": "~4.5.0",
    "babel-eslint": "^10.1.0",
    "eslint": "^8.9.0",
    "eslint-plugin-vue": "^6.2.2",
    "vue-template-compiler": "^2.6.11"
  }
`

**Comment by ddisqq** [comment]
Same issue here:
`    "devDependencies": {
        "@vue/cli-plugin-babel": "~4.5.0",
        "@vue/cli-plugin-eslint": "~4.5.0",
        "@vue/cli-service": "~4.5.0",
        "eslint-plugin-vue": "^6.2.2",
        "vue-template-compiler": "^2.6.11"
    }`

[https://drive.google.com/file/d/1OLUqy-qnydZsmKwGWFUY88vVTqo4wjRi/view?usp=sharing](url)
